csharp/private/frameworks/net20.BUILD

```diff
@@ -0,0 +1,668 @@
+# This file was generated by build\src\D2L.Build.BazelGenerator\FrameworkBuildGenerator\GenerateFrameworkBuildFile.cs
```

Comment:
Each of these was independently generated?

Reply:
Correct